src/region.js

```diff
@@ -47,36 +47,53 @@ const Region = MarionetteObject.extend({
   // method for you. Reads content directly from the `el` attribute. The `preventDestroy`
   // option can be used to prevent a view from the old view being destroyed on show.
   show(view, options) {
-    if (!this._ensureElement(options)) {
-      return;
-    }
+    if (!this._ensureElement(options)) { return this; }
```

Comment:
Seems more stylistic really.  We return this nearly everywhere for chaining purposes, but then in this case it will have thrown an error so I don't know that the return is ever really used. The only possible benefit other than just consistency I see here is that because we return this in so many places, this may gzip better?  But that could be bs.